app/controllers/catalog_controller.rb

```diff
@@ -690,17 +690,17 @@ def authorize_email_use!
       }
     end
 
-    config.add_search_field('journal title_quote',:include_in_advanced_search => false) do |field|
+    config.add_search_field('journaltitle_quote',:include_in_advanced_search => false) do |field|
       field.solr_parameters = { :'format' => "Journal" }
       field.include_in_simple_select = false
       field.solr_local_parameters = {
 #        :qf => '$title_quot_qf',
 #        :pf => '$title_quot_pf',
-        :search_field => "journal title"
+        :search_field => "journaltitle"
```

Comment:
Trying to see what this does. I don't see journaltitle in the solr schema. Where is 'journaltitile' mapped to something in solr?

Reply:
In the specs, we appear to map the journaltitle search field to title with a format of Journal/Periodical. There is one SearchBuilder spec that I noted with a TODO that we appear to use journaltitle_quoted that I thought was peculiar and could use more investigation, but these changes didn't affect that behavior.